src/index.html

```diff
@@ -7,7 +7,252 @@
+          +1 234 555 55 55
+        </section>
+        <section class="header__logoliner-2">
+          <img
+            class="header__logo"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </section>
+        <section class="header__logoliner-3">
+          <a href="#">CALL TO ORDER</a>
+        </section>
+      </article>
+      <input
+        id="menu"
+        class="menu"
+        type="checkbox"
+        hidden
+      >
+      <article class="header__logoliner-mobile">
+        <img
+          class="header__logoliner-1"
+          src="images/phone.svg"
+          alt="phone"
+        >
+        <img
+          class="header__logoliner-2"
+          src="images/logo.png"
+          alt="logo"
+        >
+        <label for="menu">
+          <img
+            class="header__logoliner-3"
+            src="images/menu.svg"
+            alt="menu"
+          >
+        </label>
+      </article>
+      <ul>
+        <li>
+          <a href="#">HOME</a>
+        </li>
+        <li>
+          <a href="#">ABOUT US</a>
+        </li>
+        <li>
+          <a href="#">GALLERY</a>
+        </li>
+        <li>
+          <a href="#">CONSULTATION</a>
+        </li>
+        <li>
+          <a href="#">TESTIMONIALS</a>
+        </li>
+        <li>
+          <a href="#">CONTACTS</a>
+        </li>
+        <li><a href="#">+ 1 234 555-55-55</a></li>
+        <li class="header__logoliner-3">
+          <a href="#">CALL TO ORDER</a>
+        </li>
+      </ul>
+      <article class="header__title">
+        <h1 class="header__text">Miami Condo Kings</h1>
+      </article>
+      <nav class="header__navigation">
+        <a class="header__navigation-link" href="#">HOME</a>
+        <a class="header__navigation-link" href="#">ABOUT US</a>
+        <a class="header__navigation-link" href="#">GALLERY</a>
+        <a class="header__navigation-link" href="#">CONSULTATION</a>
+        <a class="header__navigation-link" href="#">TESTIMONIALS</a>
+        <a class="header__navigation-link" href="#">CONTACTS</a>
+      </nav>
+    </header>
+    <main>
+      <article class="about">
+        <section class="about__left">
+          Miami best<br>Real estate <a class="blue" href="#">here</a>
+        </section>
+        <section class="about__right">
+          <strong class="about__strong">Over 50 years experience in love.</strong>
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.<br><br>
+            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.<br><br>
```

Comment:
тут має бути розрив рядка
```suggestion
            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. 
            Working directly with developers - so have more useful information about the profitable deals.<br><br>
```